recipes/repeatfinder/meta.yaml

```diff
@@ -0,0 +1,30 @@
+{% set version = 1.1 %}
```

Comment:
update this once you do a release with an actual license (https://github.com/jaroslav-kubin/repeatfinder/issues/2)